### PR TITLE
[CDAP-8424] About CDAP and Access Token pages

### PR DESCRIPTION
**Pull request by tonybach**
JIRA: https://issues.cask.co/browse/CDAP-8424
Build: http://builds.cask.co/browse/CDAP-DRC5700

### CODE REVIEW

File: cdap-ui/app/cdap/text/text-en.yaml [hunk 1]
```diff
@@ -84,6 +84,28 @@ commons:
...
+        and: " and "
+        privacyPolicy: Privacy Policy
+  AccessTokenModal:
+    modalHeader: Access Token
+    login:
+      textContent: Please enter your username and password to generate a new access token
+      usernamePlaceholder: Username
+      passwordPlaceholder: Password
+      submit: Generate access token
+      error: Login failed. Username or password incorrect.
```
Comment (bdmogal) [inline cdap-ui/app/cdap/text/text-en.yaml]: ``Login failed. Please check your username and password and try again``?

File: cdap-ui/app/cdap/text/text-en.yaml [hunk 2]
```diff
@@ -84,6 +84,28 @@ commons:
...
+    accessToken: "Your access token is: "
```
Comment (bdmogal) [inline cdap-ui/app/cdap/text/text-en.yaml]: Should this just be ``Access Token: ``?

File: cdap-ui/app/cdap/components/Header/ProductDropdown/AboutPageModal/index.js [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+              {T.translate('features.AboutPage.copyright.thirdLine.view')}
+              <a
+                href={termsConditionsUrl}
+                target="_blank"
+              >{T.translate('features.AboutPage.copyright.thirdLine.termsAndConditions')}</a>
+              {T.translate('features.AboutPage.copyright.thirdLine.and')}
+              <a
+                href={privacyPolicyUrl}
+                target="_blank"
+              >{T.translate('features.AboutPage.copyright.thirdLine.privacyPolicy')}</a>
```
Comment (elfenheart) [inline cdap-ui/app/cdap/components/Header/ProductDropdown/AboutPageModal/index.js]: nit: put the `T.translate()` on a new line

File: cdap-ui/app/cdap/components/Header/ProductDropdown/index.js [hunk 4]
```diff
@@ -70,86 +93,102 @@ export default class ProductDropdown extends Component {
...
+              onClick={this.toggleAboutPage}
             >
-              {T.translate('features.Navbar.ProductDropdown.aboutLabel')}
-            </a>
-          </DropdownItem>
-          <DropdownItem tag="ul" divider />
-          <DropdownItem tag="li">
+              <a>{T.translate('features.Navbar.ProductDropdown.aboutLabel')}</a>
+            </DropdownItem>
+            <DropdownItem tag="ul" divider />
```
Comment (elfenheart) [inline cdap-ui/app/cdap/components/Header/ProductDropdown/index.js]: why is this DropdownItem tag a `ul`?

**Comment by elfenheart** [comment]
👍  LGTM

**Comment by tonybach** [comment]
Green build, merging now.

**Pull request merged**